cmd/kubeadm/app/cmd/features.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	// alpha: v1.8
+	HA utilfeature.Feature = "HA"
```

Comment:
Can you add `SelfHosting: beta` and `StoreCertsInSecrets: alpha` instead of `HA`, please?
In a follow-up commit in this PR or a separate one, can you also remove the `selfHosting` field to respect the feature flag instead?